rs/sns/governance/src/governance/assorted_governance_tests.rs

```diff
@@ -165,12 +165,12 @@ lazy_static! {
 
     pub(crate) static ref TEST_ROOT_CANISTER_ID: CanisterId = CanisterId::from(500);
     pub(crate) static ref TEST_GOVERNANCE_CANISTER_ID: CanisterId = CanisterId::from(501);
-    static ref TEST_LEDGER_CANISTER_ID: CanisterId = CanisterId::from(502);
-    static ref TEST_SWAP_CANISTER_ID: CanisterId = CanisterId::from(503);
-    static ref TEST_ARCHIVES_CANISTER_IDS: Vec<CanisterId> =
+    pub(crate) static ref TEST_LEDGER_CANISTER_ID: CanisterId = CanisterId::from(502);
```

Comment:
I would move these so they're accessible in both, ideally, since it seems like we are finding a need for some common test code / fixtures.

Reply:
Done!